pkg/operator/operator.go

```diff
@@ -406,6 +406,20 @@ func New(
 	}
 	o.informers = append(o.informers, informer)
 
+	// Many of the cluster capabilities such as Console can be enabled after
+	// installation. So this watches for any updates to the ClusterVersion - version
```

Comment:
Maybe worth adding a comment that the console capability (like other capabilities) can only go from disabled -> enabled and not the other way around, meaning that CMO doesn't have to deal with removal of the console plugin resources.